addon/components/simple-bar.js

```diff
@@ -1,20 +1,32 @@
 import Component from '@ember/component';
-import layout from '../templates/components/simple-bar';
```

Comment:
This isn't needed, nor is the template itself.  This is because SimpleBar automatically adds its divs and classes itself.

Reply:
Interesting - didn't know about the auto-yielding-without-template behavior of components.
Is this documented somewhere?